src/Configuration/Config.php

```diff
@@ -18,17 +20,71 @@ class Config
     /** @var PathResolver */
     private $pathResolver;
 
-    public function __construct()
+    /** @var Stopwatch */
+    private $stopwatch;
+
+    /** @var FilesystemCache */
+    private $cache;
+
+    /** @var array */
+    private $timestamps = [];
+
+    public function __construct(Stopwatch $stopwatch)
     {
+        $this->stopwatch = $stopwatch;
+        $this->cache = new FilesystemCache();
         $this->pathResolver = new PathResolver(dirname(dirname(__DIR__)), []);
```

Comment:
looks like PathResolver is not needed anymore here

Reply:
Correct, will do.

Reply:
Actually, `this->config->getPath` is used in a number of places, but it should be moved out of `Config`, but that will be a separate PR.

Reply:
ok, add a todo then